README.rst

```diff
@@ -49,6 +49,7 @@ Environment Variables
 * ``ACCOUNT_PROFILE_URL`` - The URL of the account profile page.
 * ``ACCOUNT_SETTINGS_URL`` - The URL of the account settings page.
 * ``AUTHN_MINIMAL_HEADER`` - A boolean flag which hides the main menu, user menu, and logged-out
+* ``ENABLE_HEADER_WITHOUT_USERNAME`` - A boolean flag which hides the username from the header
```

Comment:
[NIT] Is it just me that the name does not look very much good? Like are not actualy Enabling the header without username? We are just hiding the username from header. Can we have a name something like `HIDE_USERNAME_IN_HEADER`?

Again, it can be just me. Please leave your opinion.

Reply:
We can use this name. I don't mind.